resources/cooked/css/common.css

```diff
@@ -696,3 +696,7 @@ p.speech-bubble::after {
     background-image: url('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBzdGFuZGFsb25lPSJubyI/PjwhRE9DVFlQRSBzdmcgUFVCTElDICItLy9XM0MvL0RURCBTVkcgMS4xLy9FTiIgImh0dHA6Ly93d3cudzMub3JnL0dyYXBoaWNzL1NWRy8xLjEvRFREL3N2ZzExLmR0ZCI+PHN2ZyB0PSIxNjMxMTA2MDI4MTg2IiBjbGFzcz0iaWNvbiIgdmlld0JveD0iMCAwIDEwMjQgMTAyNCIgdmVyc2lvbj0iMS4xIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHAtaWQ9IjIxNTgiIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB3aWR0aD0iMjAwIiBoZWlnaHQ9IjIwMCI+PGRlZnM+PHN0eWxlIHR5cGU9InRleHQvY3NzIj48L3N0eWxlPjwvZGVmcz48cGF0aCBkPSJNNTEyIDEwMjRBNTEyIDUxMiAwIDEgMSA1MTIgMGE1MTIgNTEyIDAgMCAxIDAgMTAyNHogbS0zMi4xMjgtMTkyLjM4NGM0MS4xNTIgMCA3NC41Ni0zMC41MjggNzQuNTYtNjguMTZzLTMzLjQwOC02OC4xNi03NC41Ni02OC4xNi03NC41NiAzMC41MjgtNzQuNTYgNjguMTYgMzMuMzQ0IDY4LjE2IDc0LjU2IDY4LjE2eiBtMC02MzkuMjMyYy0xMjMuNTIgMC0yMjMuNjggOTEuNTg0LTIyMy42OCAyMDQuNDhIMzY4YzAtNTYuNDQ4IDUwLjA0OC0xMDIuMjcyIDExMS44MDgtMTAyLjI3MiA2MS43NiAwIDExMS44MDggNDUuNzYgMTExLjgwOCAxMDIuMjcyIDAgMzguODQ4LTIzLjc0NCA3Mi42NC01OC42MjQgODkuOTg0YTIyNC4yNTYgMjI0LjI1NiAwIDAgMC00My4xMzYgMzAuNzg0bC0xLjE1MiAxLjAyNGExNjkuMTUyIDE2OS4xNTIgMCAwIDAtNi4wMTYgNS43NmwtMS4yMTYgMS4yMTYtMS42IDEuNnYwLjEyOGEyMDEuNjY0IDIwMS42NjQgMCAwIDAtNDQuMzUyIDcwLjAxNiAxOTAuMDggMTkwLjA4IDAgMCAwLTEwLjU2IDUyLjk5MmgtMS4wMjRsLTAuMTI4IDEyLjI4OGMwLTQuMTYgMC44MzItOC4xMjggMS4wODgtMTIuMjg4aDExMC44NDhsLTAuMDY0IDEyLjI4OGMwLTM4Ljg0OCAyMy42OC03Mi42NCA1OC41Ni04OS45MiAxOS45NjgtMTAuODggMzcuOTUyLTI0LjU3NiA1My4yNDgtNDAuNDQ4di0wLjA2NGExOTQuMDQ4IDE5NC4wNDggMCAwIDAgNTYtMTM1LjI5NmMwLjA2NC0xMTIuOTYtMTAwLjA5Ni0yMDQuNTQ0LTIyMy42MTYtMjA0LjU0NHoiIHAtaWQ9IjIxNTkiPjwvcGF0aD48L3N2Zz4=');
     background-size: 100% 100%;
 }
+
+.mission-modal-tutorial-content-right img{
```

Comment:
This is already fixed in my local development.

Reply:
All... right. I am wondering why so many commits here...